e2e/container/run_test.go

```diff
@@ -24,6 +24,11 @@ func TestRunAttachedFromRemoteImageAndRemove(t *testing.T) {
 	golden.Assert(t, result.Stderr(), "run-attached-from-remote-and-remove.golden")
 }
 
+func TestForceRemoveImage(t *testing.T) {
```

Comment:
We already have test coverage for that here in the unit test suite: https://github.com/docker/cli/blob/master/cli/command/image/remove_test.go#L71-L78

Following https://github.com/docker/cli/blob/master/TESTING.md#guidelines I don't think we need an e2e test for the force flag.

I unit test for a successful `docker rmi` would be good, but it would go in the  `e2e/image` package.